x/bucket/internal/ipfs_write_bucket_item.go

```diff
@@ -0,0 +1,19 @@
+package internal
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	shell "github.com/ipfs/go-ipfs-api"
+	bt "github.com/sonr-io/sonr/x/bucket/types"
+)
+
+func WriteBucketItem(sh *shell.Shell, b *bt.Bucket, address string, item bt.BucketItem) (string, error) {
+	itemPath := b.GetPath(address, item.Name())
+	err := sh.FilesWrite(context.Background(), itemPath, bytes.NewReader(item.Content()), shell.FilesWrite.Create(true))
```

Comment:
standardiznig the encoding on the `item.Content` will help with retrieval

Reply:
SchemaDocuments only for now? @joshLong145